.rubocop.yml

```diff
@@ -1,8 +1,13 @@
 AllCops:
   Exclude:
-    - 'db/**/*'
```

Comment:
It looks like what you did here was make the Rubocop exclusions a bit tighter in the db/ folder. Am I reading that right?

Reply:
After this change, files in `db/**/*` are no longer exempt from RuboCop's _all-seeing eye_. But, looking at lines 3 and 4 below, we want that eye to ignore migrations and the schema file.

Specific cop exclusions are set later on in this file (e.g. `Metrics/AbcSize`). The goal is to encourage authors to keep the new `*.seeds.rb` files neat and tidy without being too overbearing.